src/libcore/mem.rs

```diff
@@ -216,23 +298,31 @@ pub fn align_of_val<T: ?Sized>(val: &T) -> usize {
     unsafe { intrinsics::min_align_of_val(val) }
 }
 
-/// Creates a value initialized to zero.
+/// Creates a value whose bytes are all zero.
```

Comment:
"Whose" is fine. English has "who" and "which", but when made possessive both effectively become "whose". I think "zero" and "zeros" are both fine. "All zeros" makes me think "each bit of the byte is zero", and "all zero" makes me think "each byte of the value has a value of zero".